packages/eslint-config-hypothesis/ts.js

```diff
@@ -0,0 +1,33 @@
+import tseslint from 'typescript-eslint';
+import base from './base.js';
+
+export default tseslint.config(
+  ...base,
+  ...tseslint.configs.recommended,
+  {
+    rules: {
+      // Prop validation is performed by TypeScript + JSDoc.
+      'react/prop-types': 'off',
```

Comment:
I need to check if defining this rule here would cause an error when react plugin is not enabled.

Reply:
We could move this rule into the React configuration. There are no projects where we are using Preact but not using TypeScript (either directly or via JSDoc), so we can assume that any project using Preact is not using prop-types.

Reply:
> either directly or via JSDoc

Hah, I didn't think about this point. I think I'll disable it then.